### ISSUE TITLE
Testing out evolving small variant callers

### ISSUE THREAD

**Issue by SHuang-Broad** [broadinstitute/long-read-pipelines#181]
Small variant callers with LRs are being pushed out/updated, so we need to keep track of major updates and new comers

List of interesting ones

New
- [ ] [dipcall](https://github.com/lh3/dipcall)
- [ ] [NanoCaller](https://github.com/WGLab/NanoCaller)
- [ ] [PEPPER](https://github.com/kishwarshafin/pepper) (Maybe? considering it is much more than a variant caller)

Track major updates of 

* [DV](https://github.com/google/deepvariant)
* [Longshot](https://github.com/pjedge/longshot)
* [Clair](https://github.com/HKU-BAL/Clair)

**Comment by kvg** [comment]
I second this.  I'll also suggest that we add [noise-cancelling repeat finder](https://github.com/makovalab-psu/NoiseCancellingRepeatFinder) (NCRF) to the mix.

**Comment by SHuang-Broad** [comment]
I think NCRF deserves a separate ticket tracking STR callers?

**Comment by kvg** [comment]
Yeah, I've looked into NCRF and am no longer sure it'll easily go into our workflows.

**Comment by SHuang-Broad** [comment]
Given that Clair3 doesn't seem to [plan](https://github.com/HKU-BAL/Clair3/issues/74) supporting CLR data (the claim there is interesting), we might be forced to evaluate NanoCaller, which calls InDels for CLR data.
